# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/l10n/DocumentL10n.h
# Commit: 3404fd6725f1
# Full Hash: 3404fd6725f15d1253a7f76e3937185bd353f263
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-02 21:43:56
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part2: Switch Localization class to use localization-ffi. r=emilio,nika
#   
#   Depends on D104788
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104789
# ==============================================================================

diff -r c9cfbba15d90 -r 3404fd6725f1 dom/l10n/DocumentL10n.h
--- a/dom/l10n/DocumentL10n.h	Mon Aug 02 09:54:07 2021 +0000
+++ b/dom/l10n/DocumentL10n.h	Mon Aug 02 09:54:07 2021 +0000
@@ -47,12 +47,10 @@
   NS_DECL_ISUPPORTS_INHERITED
   NS_DECL_CYCLE_COLLECTION_CLASS_INHERITED(DocumentL10n, DOMLocalization)
 
-  static RefPtr<DocumentL10n> Create(Document* aDocument, const bool aSync);
+  static RefPtr<DocumentL10n> Create(Document* aDocument, bool aSync);
 
  protected:
-  explicit DocumentL10n(Document* aDocument, const bool aSync);
-  bool Init() override;
-
+  explicit DocumentL10n(Document* aDocument, bool aSync);
   virtual ~DocumentL10n() = default;
 
   RefPtr<Document> mDocument;